service/ztests/accept-header.yaml

```
@@ -0,0 +1,22 @@
+script: |
+  source service.sh
+  zed create -q test
+  echo '{ts:0}' | zed load -use test -q -
+  for accept in "text/plain, application/json" "application/xml, text/css"; do
+    echo === $accept ===
+    curl -X POST \
+      -H "Accept: $accept" \
+      -d '{"query":"from test@main"}' \
+      $ZED_LAKE/query
```

`curl -d` implies `-X POST`.
```suggestion
    curl -H "Accept: $accept" -d '{"query":"from test@main"}' $ZED_LAKE/query
```

---

The more you know